src/ui/components/label.c

```diff
@@ -175,6 +177,28 @@ static void _cleanup(component_t* component)
     ui_util_component_cleanup(component);
 }
 
+void _measure_label_dimensions(component_t* label)
+{
+    data_t* data = (data_t*)label->data;
+
+    UG_FontSetVSpace(2);
+    UG_FontSelect(data->font);
+    if (data->scrollable) {
+        UG_MeasureStringNoBreak(&(label->dimension.width), &(label->dimension.height), data->text);
+        if (label->dimension.width < SCREEN_WIDTH) {
+            // Do not scroll if text already fits in the screen
+            data->scrollable = false;
+        }
+    } else if (
```

Comment:
else if has no condition?

Reply:
I just moved the code. I'll fix it

Reply:
I think you are mis-seeing a left parenthesis as a left bracket.